website/docs/api/kb.md

```diff
@@ -4,169 +4,88 @@ teaser:
   A storage class for entities and aliases of a specific knowledge base
   (ontology)
 tag: class
-source: spacy/kb.pyx
+source: spacy/kb/kb.pyx
 new: 2.2
 ---
 
-The `KnowledgeBase` object provides a method to generate
-[`Candidate`](/api/kb/#candidate) objects, which are plausible external
-identifiers given a certain textual mention. Each such `Candidate` holds
-information from the relevant KB entities, such as its frequency in text and
-possible aliases. Each entity in the knowledge base also has a pretrained entity
-vector of a fixed size.
+The `KnowledgeBase` object is an abstract class providing a method to generate
+[`Candidate`](/api/candidate) objects, which are plausible external identifiers
+given a certain textual mention. Each such `Candidate` holds information from
+the relevant KB entities, such as its frequency in text and possible aliases.
+Each entity in the knowledge base also has a pretrained entity vector of a fixed
+size.
+
```

Comment:
We should have an infobox here that explains that this same class was not abstract up until spaCy version X

Reply:
Added a short disclaimer. Feel free to resolve this thread if you think it's sufficient.